DSL/DataMapper/controllers/mapper.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import Handlebars from 'handlebars';
+import fs from "fs";
+import helpers from "../lib/helpers.js";
+
+// For running on docker:
+const pathToHbs = '/usr/src/app/views/training/'
+
+// For running on local:
+// const pathToHbs = "DSL/DataMapper/views/training/";
```

Comment:
You should be able to use docker volume(s) to use same settings for running it in docker or local.

Reply:
Docker settings will be fixed in a future commit